helpers/html_converter.sh

```diff
@@ -0,0 +1,248 @@
+#!/bin/bash
+
+# emba - EMBEDDED LINUX ANALYZER
+#
+# Copyright 2020-2021 Siemens AG
+#
+# emba comes with ABSOLUTELY NO WARRANTY. This is free software, and you are
+# welcome to redistribute it under the terms of the GNU General Public License.
+# See LICENSE file for usage of this software.
+#
+# emba is licensed under GPLv3
+#
+# Author(s): Stefan Haboeck
+
+declare -a MENU_LIST
+
+build_index_file(){
+  
+  local TOP10_FORMAT_COUNTER
+  local HTML_FILE
+  FILE=$1
+  FILENAME=$(basename "$FILE")
+  HTML_FILE="$(basename "${FILE%.txt}"".html")"
+  COULORLESS_FILE_LINE=$(head -1 "$FILE" | tail -n 1 | cut -c27-)
```

Comment:
COLORLESS_...

Reply:
Down below you used `COLORLESS_FILE_LINE` < it seems you forgot to search for occurrences of this variable and used the correct name everywhere.

Reply:
Line 177: ` 	COLORLESS_FILE_LINE=${FILE_LINE:26:${#FILE_LINE}-3}	`